zipkin-thrift/build.gradle

```diff
@@ -0,0 +1,11 @@
+buildscript {
+	repositories {
+		jcenter()
+	}
+
+	dependencies {
+		classpath griddle
+	}
+}
+
+apply plugin: 'idl'
```

Comment:
This `buildscript {}` block is the example same as the one in `zipkin-scrooge` but this one applies the 'idl' plugin, and the other applies the 'scrooge' plugin.

Are both plugins defined in this `griddle` dep?